src/database/schema/ChannelListMemberSchema.ts

```diff
@@ -102,6 +102,10 @@ class ChannelListMemberSchema implements BaseDbSchema {
             B.bio,
             B.is_banned,
             B.last_active_at,
+            B.anon_user_info_color_code,
+            B.anon_user_info_color_name,
+            B.anon_user_info_emoji_code,
+            B.anon_user_info_emoji_name,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new fields `anon_user_info_color_code`, `anon_user_info_color_name`, `anon_user_info_emoji_code`, and `anon_user_info_emoji_name` have been added to the SQL query. Ensure that these fields exist in your database and are correctly populated. Also, consider handling potential null values for these fields in the application logic where this data is used. If these fields can be null, you should add null checks or default values where necessary to prevent potential runtime errors.


<!-- This is an auto-generated comment by OSS CodeRabbit -->